.all-contributorsrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": ["readme.md"],
+  "imageSize": 100,
```

Comment:
We can adjust how big the avatars should be in the table, size is in pixels. This is the default setting.